components/openvsx-proxy/BUILD.yaml

```diff
@@ -16,6 +16,7 @@ packages:
       - GOOS=linux
     config:
       packaging: app
+      buildCommand: ["go", "build", "-trimpath", "-ldflags", "-buildid= -w -s -X 'github.com/gitpod-io/gitpod/openvsx-proxy/cmd.Version=commit-${__git_commit}'"]
```

Comment:
We don't have a `cmd.Version` variable in the code but I think this is fine since this is simply ignored, right?

Reply:
it is indeed.